pkg/controller/masteruserrecord/sync_test.go

```diff
@@ -130,7 +130,7 @@ func TestSyncMurStatusWithUserAccountStatusWhenCompleted(t *testing.T) {
 func TestSynchronizeUserAccountFailed(t *testing.T) {
 	// given
 	l := logf.ZapLogger(true)
```

Comment:
I'm not sure if `l` is still used. Also, you could keep those 2 lines as `logf.SetLogger(logf.ZapLogger(true))`.

Reply:
@xcoulon i think it's used here: https://github.com/codeready-toolchain/host-operator/pull/146/files#diff-37787ca0ef3b20842b503a57aeae391fR152 looks like we might need to keep it